src/content/developers/docs/consensus-protocols/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Consensus protocols
+description: An explanation of consensus protocols in distributed systems and the role they play in Ethereum.
+lang: en
+sidebar: true
+incomplete: true
+---
+
+A fundamental problem in distributed computing is to achieve overall system reliability while individual nodes of the network may be faulty, malicious or otherwise unreliable.
+
+When it comes to blockchains like Ethereum, which are in essence distributed databases, the nodes of the network must be able to reach agreement on the current state of the system. This is achieved using consensus protocols.
+
+## Prerequisites {#prerequisites}
+
+To better understand this page, we recommend you first read our [introduction to Ethereum](/developers/docs/intro-to-ethereum/).
+
+## What is a consensus protocol?
+
+Consensus protocols (also known as consensus mechanisms or consensus algorithms) allow distributed systems (networks of computers) to work together.
+
+For decades, consensus protocols have been used to establish consensus among database nodes, application servers, and other enterprise infrastructure. In recent years, new consensus protocols have been invented to allow cryptoeconomic systems, such as Ethereum, to agree on the state of the network.
+
+Using a consensus protocol within Ethereum allows the nodes of the Ethereum network to agree on the state of all information recorded on the Ethereum blockchain, and prevents certain kinds of economic attacks.
+
+<!-- ### Consensus -->
+
+<!-- Formal requirements for a consensus protocol may include: -->
+
+<!-- - Agreement: All correct processes must agree on the same value. -->
+<!-- - Weak validity: For each correct process, its output must be the input of some correct process. -->
+<!-- - Strong validity: If all correct processes receive the same input value, then they must all output that value. -->
+<!-- - Termination: All processes must eventually decide on an output value -->
+
+<!-- ### Fault tolerance -->
+<!-- TODO explain how protocols must be fault tolerant -->
+
+## Types of consensus protocols
+
+<!-- TODO -->
+<!-- Why do different consensus protocols exist? -->
+<!-- What are the tradeoffs of each? -->
+
+### Proof of work
+
+Ethereum, like Bitcoin, currently uses a [proof-of-work (PoW)](/developers/docs/consensus-protocols/pow/) consensus protocol.
```

Comment:
A rule on hyphenation.

Hyphenate proof-of-work when used in the middle of a sentence.
You don't need to hyphenate it when it comes at the end of a sentence like proof of work. 
It also doesn't need to be hyphenated in a title or when used in isolation like in a list.

Reply:
Interesting! Good to know.

There also appears to be different naming styles across the internet.

e.g. https://consensys.net/blog/blockchain-explained/what-is-proof-of-stake/
Here they always capitalize (Proof of Work & Proof of Stake)

e.g. https://en.wikipedia.org/wiki/Proof_of_stake
Seems to keep things lowercase

I default to you on this, just want us to be consistent.